### PR TITLE
Update support libraries to 25.3.1 and update gradle-witness hashes

### PR DESCRIPTION
**Pull request by hagau**
## Description
Updated support libraries and ran ./gradlew calculateChecksums

## How Has This Been Tested?
Compiles just fine. Deployment to test device apparently needs multidex.

### CODE REVIEW

**Comment by dschuermann** [comment]
Can you enable multidex in this PR for debug builds only?

File: OpenKeychain/build.gradle [hunk 1]
```diff
@@ -15,6 +15,7 @@ dependencies {
     compile 'com.android.support:recyclerview-v7:25.3.1'
     compile 'com.android.support:cardview-v7:25.3.1'
     compile 'com.android.support:support-annotations:25.3.1'
+    compile 'com.android.support:multidex:1.0.1'
```
Comment (Valodim) [inline OpenKeychain/build.gradle]: this is pointless if we don't inherit our application class from `MultiDexApplication`. it's also only needed to build with multidex for minsdk 20 or lower, so we can just drop this.

Comment (hagau) [reply OpenKeychain/build.gradle]: I apparently did not read the documentation thoroughly enough, thanks.

**Comment by hagau** [comment]
This now passes all tests.

**Pull request merged**